chipy_org/apps/meetings/models.py

```diff
@@ -117,6 +121,11 @@ def __str__(self):
     )
     description = tinymce_models.HTMLField(blank=True, null=True)
 
+    in_person_capacity = models.PositiveSmallIntegerField(null=False)
```

Comment:
Having done this for a long time I'm torn on the in_person_capacity not having a default. I think that might make sense...but honestly setting it to something low might make more sense. Like setting it to 50 or null by default would reduce a barrier to creating these events. You might publish events 1 week later because it will take time to figure out what the capacity is, which I would think is worse than just setting a number that is a small default and raising it over time.

That said, I can see the case to leave it like this.

Reply:
I could go either way on the default size.  I think the bigger issue is changing the in_person_capacity after the fact regardless.  I need to capture a post_save signal on meeting and adjust the wait list based on the new capacity.